internal/cmd/ktf/init.go

```diff
@@ -1,4 +1,4 @@
-package cmd
+package ktf
```

Comment:
https://github.com/Kong/kubernetes-testing-framework/pull/58#discussion_r675822671 

no action required, just having a strong feeling of "I told you so" :joy:

Reply:
ha